src/core/msg.rs

```diff
@@ -86,6 +114,7 @@ impl ValidatedMsg for ExecuteMsg {
                 payable_type,
                 payable_uuid,
                 scope_id,
+                oracle_address,
```

Comment:
Payable type was already a factor in the "register payable" flow simply for validation, but now it's actually important.  But anyway, that's why I only needed to add oracle address and not both.  Now they're tied to the individual payable instead of the contract itself, and that's nice